### PR TITLE
Modification of the summary endpoint scraping test to check for corre…

### PR DESCRIPTION
**Pull request by dnavre**
Did this to find a bug with Heapster not reporting pod volume usage data even though it's available in the Kublet API. Issue #1719.

The test case actually proved that the suspect code actually works correctly and the issue is probably somewhere else.

### CODE REVIEW

**Comment by DirectXMan12** [comment]
I'm fine with testing the volume scraping logic, but can you adapt this to just work in the main decoding test?

**Comment by dnavre** [comment]
@DirectXMan12 , as requested I moved the test logic to the decoding test case. :)

File: metrics/sources/summary/summary_test.go [hunk 1]
```diff
@@ -109,6 +118,7 @@ func testingSummaryMetricsSource() *summaryMetricsSource {
 }
 
 func TestDecodeSummaryMetrics(t *testing.T) {
+
```
Comment (piosz) [inline metrics/sources/summary/summary_test.go]: remove empty line

**Comment by piosz** [comment]
/lgtm
Just one minor comment. Please squash commits before merge.

**Comment by dnavre** [comment]
Squashed commits. Looks like there's nothing else to do, right? )

**Comment by DirectXMan12** [comment]
LGTM.  merging.

**Pull request merged**